### PR TITLE
Changes to combat borg disponibility

### PR DESCRIPTION
**Pull request by Alberyk**
Locks combat borg behind an extra keycard swipe thing, and a research upgrade, but, they can still be enacted via code delta or admin bus.

### CODE REVIEW

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 1]
```diff
@@ -233,7 +233,7 @@
 		return
 	var/list/modules = list()
 	modules.Add(robot_module_types)
-	if(security_level == (SEC_LEVEL_RED || SEC_LEVEL_DELTA)) //There are no brakes in the combat module train.
+	if((crisis_override && security_level == SEC_LEVEL_RED) || SEC_LEVEL_DELTA) //no fun allowed anymore.
```
Comment (Bedshaped) [inline code/modules/mob/living/silicon/robot/robot.dm]: `if ((condition AND condition) OR TRUE)` someone correct me if I'm wrong but this statement will always pass without fail.

I think it needs to be `if((crisis_override && security_level == SEC_LEVEL_RED) || security_level == SEC_LEVEL_DELTA)`

**Comment by Alberyk** [comment]
Changes done.

**Comment by skull132** [comment]
Can crisis override be set by commandstaff? I don't think it can, but it should be.

**Comment by Alberyk** [comment]
I don't wish for so, because, it would end in the same situation with code red. The purpose is to make combat borgs more rare, as in, they will depends on some resources to even appear, just putting an extra swipe or something is not the same.

**Comment by skull132** [comment]
That's unproven, however.

Consider: the **proposal and request** for such a check condition from regulars of Command Staff. This is not an immense nor immediate issue. We can approach it incrementally. If another swipe doesn't resolve the problem, then add another condition or block.

**Pull request merged**